core/src/script/CGXP/plugins/Editing.js

```diff
@@ -485,6 +492,57 @@ cgxp.plugins.Editing = Ext.extend(gxp.plugins.Tool, {
         this.initKeyEvent();
     },
 
+    /** private: method[applyState]
+     */
+    applyState: function(state) {
+        if (state.layer !== undefined) {
+            if (state.id !== undefined) {
+                var protocol = new OpenLayers.Protocol.HTTP({
+                    url: this.layersURL + state.layer,
```

Comment:
We should read only one feature: http://trac.mapfish.org/trac/mapfish/wiki/MapFishProtocol#Readonefeature